### ISSUE TITLE
Elevated plasma tetradecanolycarnitine, C14:0

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5971]
**Preferred term label:** Elevated plasma tetradecanolycarnitine, C14:0

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Tetradecanolycarnitine concentration in the blood.

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I am going to add one term for each of the tetradecanolycarnitine new term requests -- I think that the C14:0 etc are measurement artefacts and that we can map all of these tests to the same term. If this is incorrect, please reopen and provide more context/information.
